src/Converter.php

```diff
@@ -179,11 +179,12 @@ public static function convert($opt) {
         $context = self::createContext($opt);
 
         $result = file_get_contents($endpoint, false, $context);
-        if (substr($http_response_header[0], 9, 3) !== '200') { //Check http response code for if the request failed
+         $http_response = substr($http_response_header[0], 9, 3);
```

Comment:
There's an extra space here
```suggestion
        $http_response = substr($http_response_header[0], 9, 3);
```